src/App.tsx

```diff
@@ -15,71 +17,94 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface Button {
+  data: string
+  class: string
+  action: () => void
+  symbol: string
+}
+
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+
+  const clearSelected = () => setSelectedGood('');
+  const isSelected = (good: string) => selectedGood === good;
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const getButtonProps = (good: string): Button => ({
```

Comment:
wow, metabutton 😄

Reply:
I had to google that, but yeah exactly 😅